src/controllers/Auth.controller.ts

```diff
@@ -114,7 +134,7 @@ export const resetPasswordRequestController = async (
       .status(404)
       .json({ status: 404, success: false, message: "Email does not exist" });
   } else {
-    const resetToken = await generateToken(user.dataValues.email, "5m");
+    const resetToken = await generateToken(user.dataValues, "5m");
```

Comment:
I think this is contrary to what we discussed.

I don't think you are supposed to sign the entire user's data inside the token.

Just the `email` or `id` is enough

Reply:
You are still setting the entire user object on the token? Is there a specific reason why you are doing this?